### ISSUE TITLE
Document the use of DOTENV to load env variables

### ISSUE THREAD

**Issue by maurodoglio** [jazzband/django-configurations#165]
The documentation only has instructions for envdir, I didn't find any mention to the use of DOTENV.

**Comment by bittner** [comment]
Oh, cool, that's supported!

Looks like a class attribute `DOTENV` needs to be [set to the file path](https://github.com/jazzband/django-configurations/blob/master/configurations/base.py#L80), then that file will be parsed and its values pushed into the process environment (just like [django-environ](https://django-environ.readthedocs.io/en/latest/)'s `.env` file by default), sweet! This happens automatically via [pre_setup](https://github.com/jazzband/django-configurations/blob/master/configurations/base.py#L111).

Related changes: PR #60.

Proposed documentation
--------------
Configurations can read values for environment variables out of an `.env` file, and push them into the application's process environment. Simply set the `DOTENV` setting to the appropriate file name: 
``` python
# mysite/settings.py

from os.path import dirname, join
from configurations import Configuration

BASE_DIR = dirname(dirname(__file__))

class Dev(Configuration):
    DOTENV = join(BASE_PATH, '.env')

    SECRET_KEY = values.SecretValue()
```
An `.env` file is an `.ini`-style file. It must contain a list of `KEY=value` pairs, just like Shell environment variables:
``` ini
# .env

DJANGO_DEBUG=False
DJANGO_SECRET_KEY=1q2w3e4r5t6z7u8i9o0(%&)$§"!pqaycz
```

**Comment by bittner** [comment]
Fixed with #223 via f38e07b94a.
